src/pathfinder/water_regions.cpp

```diff
@@ -50,7 +50,7 @@ class WaterRegionData {
 	std::array<TWaterRegionTraversabilityBits, DIAGDIR_END> edge_traversability_bits{};
 	std::unique_ptr<TWaterRegionPatchLabelArray> tile_patch_labels; // Tile patch labels, this may be nullptr in the following trivial cases: region is invalid, region is only land (0 patches), region is only water (1 patch)
 	bool has_cross_region_aqueducts = false;
-	TWaterRegionPatchLabel number_of_patches = 0; // 0 = no water, 1 = one single patch of water, etc...
+	TWaterRegionPatchLabel number_of_patches{0}; // 0 = no water, 1 = one single patch of water, etc...
```

Comment:
Hmm, semantics... is number of patches actually a `TWaterRegionPatchLabel`?

Reply:
You are right, it is not. It just needs to be the same underlying type. Easy enough to change.